zzzz_modular_occulus/code/game/jobs/job/guild.dm

```diff
@@ -15,3 +15,7 @@
 		STAT_TGH = 10,
 		STAT_VIG = 10
 	)
+	perks = list(/datum/perk/deep_connection, /datum/perk/oddity/market_prof)
```

Comment:
Union Techs and Miners shouldn't start with this perk.
We want to add a soulcrypt program for the export scanner.